app/src/main/res/values/styles.xml

```diff
@@ -5,6 +5,17 @@
         <item name="colorPrimary">@color/pumpkin_primary</item>
         <item name="colorPrimaryDark">@color/pumpkin_primaryDark</item>
         <item name="colorAccent">@color/pumpkin_accent</item>
+        <item name="title_color">@color/grey_1000b</item>
+        <item name="dim_title_color">@color/grey_500</item>
+        <item name="list_bg_color">@color/grey_1000w</item>
+        <item name="card_style">@style/CardView</item>
+    </style>
+
+    <style name="DarkPumpkinTheme" parent="Theme.AppCompat.NoActionBar">
+        <item name="title_color">@color/grey_1000w</item>
```

Comment:
Why doesn't DarkPumpkinTheme have item named `colorPrimary` and `colorAccent` like PumpkinTheme?